### PR TITLE
User notifications

### PR DESCRIPTION
**Pull request by OksanaFilipchuk**
# User notifications page and pop-up

## Summary Of Changes :fire:
The pop-up and page with user's notifications were created 

## Added
* menu-item "Notifications"
* pop-up component with user's notifications(without data)
* page with user's notifications(without data)

## Changed
* styles

## How to test :clipboard:

# PR Checklist Forms

_(to be filled out by PR submitter)_
- [ ] Code is up-to-date with the `dev` branch.
- [x] You've successfully built and run the tests locally.
- [x] There are new or updated unit tests validating the change.
- [x] JIRA/ Github Issue number & title in PR title (ISSUE-XXXX: Ticket title)
- [x] This template filled (above this section).
- [x] Sonar's report does not contain bugs, vulnerabilities, security issues, code smells ar duplication
- [x] `NEED_REVIEW` and `READY_FOR_REVIEW` labels are added.
- [x] All files reviewed before sending to reviewers

### CODE REVIEW

File: src/app/main/component/user/components/profile/user-notifications/user-notifications-pop-up/user-notifications-pop-up.component.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<div
+  [ngClass]="{ 'pop-up-container': notificationToShow.length, 'pop-up-container-empty': !notificationToShow.length }"
```
Comment (katia9261) [inline src/app/main/component/user/components/profile/user-notifications/user-notifications-pop-up/user-notifications-pop-up.component.html]: You can use here `"notificationToShow.length ? 'pop-up-container' : 'pop-up-container-empty'"`

File: src/app/main/component/user/components/profile/user-notifications/user-notifications-pop-up/user-notifications-pop-up.component.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <mat-divider></mat-divider>
+  <div *ngIf="!notificationToShow.length" class="no-notifications d-flex justify-content-center">
+    <p class="body-2 text-center">{{ 'homepage.notifications.no-new-notifications' | translate }}</p>
+  </div>
+
+  <div *ngIf="notificationToShow.length">
+    <mat-accordion class="list-wrapper" multi="false" [togglePosition]="'before'">
+      <mat-expansion-panel
+        class="notification"
+        [ngClass]="{
```
Comment (katia9261) [inline src/app/main/component/user/components/profile/user-notifications/user-notifications-pop-up/user-notifications-pop-up.component.html]: same logic here `[ngClass]="notification.read ? 'notification-read' : 'notification-unread'"`

File: src/app/main/component/user/components/profile/user-notifications/user-notifications.component.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<div class="container" aria-label="user-notification">
+  <h1 class="heading h5">{{ 'homepage.notifications.title' | translate }}</h1>
+  <div *ngIf="notifications.length" class="filter-search-wrapper d-flex">
+    <div class="filter-wrapper d-flex">
+      <span class="filter-header h6">Filter by</span>
```
Comment (katia9261) [inline src/app/main/component/user/components/profile/user-notifications/user-notifications.component.html]: Don't forget to add here translation for `Filter by`

File: src/app/main/component/user/components/profile/user-notifications/user-notifications.component.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      <div class="filter-button">
+        <button class="global-tag">All</button>
```
Comment (katia9261) [inline src/app/main/component/user/components/profile/user-notifications/user-notifications.component.html]: The same) And also for placeholder on line 15

File: src/app/main/component/user/components/profile/user-notifications/user-notifications.component.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        <button *ngFor="let filter of projects" class="tag-button">
+          <a class="custom-chip global-tag" [ngClass]="{ 'global-tag-clicked': true }">
```
Comment (katia9261) [inline src/app/main/component/user/components/profile/user-notifications/user-notifications.component.html]: If you're setting the global-tag-clicked class to true, there's no need to use a binding at all. Just `<a class="custom-chip global-tag global-tag-clicked">`

Comment (OksanaFilipchuk) [reply src/app/main/component/user/components/profile/user-notifications/user-notifications.component.html]: Of course. It was added to see how it looks. I removed this class for now. I'll  implement it  when I add a filter

**Review by mdashko — APPROVED** [review]

**Review by NadiiaStoiko — APPROVED** [review]

**Review by sobkiv — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**